### DISCUSSION TITLE
How to disable MFA

### DISCUSSION THREAD

**Discussion by Oryon007** [urania-dev/snapp#72] [Q&A]
Hi, 

I recently created a Docker container for Snapp and went through the process of logging in and getting it setup. However, when I was setting up the MFA for the admin account, my browser crashed before I could capture the QR code. When I went back to try to log into the account, it's now prompting for the code that I don't have. Is there a way to reset the MFA without having to start over?

Thanks!

**Comment by urania-dev** [comment]
Atm i think the only way would be accessing through db 🤔 and deleting user crypted mfa code

**Comment by urania-dev** [comment]
New version has more reactive and intuitive mfa enabling method with failsafe that should allow to set a MFA and enable it per app or per user preference
